components/xsd-fu/templates-java/MetadataStore.template

```diff
@@ -144,46 +166,111 @@ import java.util.Map;
  */
 public interface MetadataStore extends BaseMetadata
 {
+        /**
+         * Create root node.  The action taken here is specific to the
+         * concrete metadata implementation.
+         */
 	void createRoot();
 
+        /**
+         * Get the root node of the metadata.  Note that the root node
+         * type will be specific to the concrete metadata
+         * implementation.
+         *
+         * @return the root node.
+         */
 	MetadataRoot getRoot();
 
+        /**
+         * Set the root node of the metadata.  Note that the root node
+         * type will be specific to the concrete metadata
+         * implementation.  An exception will be thrown if the root
+         * node is of an incompatible type.
+         *
+         * @param root the root node.
+         */
 	void setRoot(MetadataRoot root);
 
 	// -- Entity storage (manual definitions) --
 
+        /**
+         * Set the BigEndian property of Pixels.
+         *
+         * @param bigEndian @c true if big endian, or @c false if little endian.
```

Comment:
This does not render as expected in the Javadoc - see <a href="https://ci.openmicroscopy.org/job/BIOFORMATS-5.1-merge-build/javadoc/ome/xml/meta/MetadataStore.html#setPixelsBinDataBigEndian(java.lang.Boolean,%20int,%20int)">setPixelsBinDataBigEndian</a>. Use `{@code }` instead?